yamlfix.toml

```diff
@@ -0,0 +1,5 @@
+explicit_start=false
```

Comment:
@mikealfare we can configure what we want here. The default currently enforces a character limit per line which is what a lot of these changes are. Before I would merge any of these, I would want to test each workflow to make sure the line break doesn't mess anything up. I would also argue that we should format quotes to all be the same but that would also require testing the workflows as well to make sure that doesn't change behavior.